cheroot/ssl/__init__.py

```diff
@@ -48,6 +49,11 @@ def bind(self, sock):
         )
         return sock
 
+    def prompt_for_tls_password(self) -> str:
```

Comment:
Let's avoid making this a public method for now:
```suggestion
    def _prompt_for_tls_password(self) -> str:
```
Also, another in-progress PR (#800) is putting another helper at the end so this should probably be the last definition too.